haystack_experimental/chat_message_stores/in_memory/chat_message_store.py

```diff
@@ -0,0 +1,86 @@
+# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
```

Comment:
Why do we need an extra `in_memory` submodule? I'd remove it.

Reply:
Yes, will do